lib/sda/protocol.go

```diff
@@ -24,17 +25,13 @@ type ProtocolInstance interface {
 	// Start on it.
 	Start() error
 	// Dispatch is called at the beginning by SDA for listening on the channels
-	// XXX Should remove that => not using ncessarily channels
```

Comment:
Why remove the comment ?? It's still accurate, we don't necessarily uses chanels, so why should we have a dispatch method ?